safenode/src/bin/cli/register.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use safenode::client::{Client, Error as ClientError};
+
+use clap::Subcommand;
+use eyre::Result;
+use xor_name::XorName;
+
+#[derive(Subcommand, Debug)]
+pub enum RegisterCmds {
+    /// Create a new register with the given pet name.
+    Create {
+        #[clap(name = "name")]
+        name: String,
+    },
+    Edit {
+        #[clap(name = "name")]
+        name: String,
+        ///
```

Comment:
We should get into the habit early of properly documenting every command to make the CLI look professional.

Reply:
Definitely. I will fix.